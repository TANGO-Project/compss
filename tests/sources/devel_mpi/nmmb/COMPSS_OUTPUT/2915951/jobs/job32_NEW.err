[BINARY EXECUTION WRAPPER] ------------------------------------
[BINARY EXECUTION WRAPPER] CMD ERROR:
ifort: command line warning #10212: -fp-model precise evaluates in source precision with Fortran.
[BINARY EXECUTION WRAPPER] ------------------------------------
