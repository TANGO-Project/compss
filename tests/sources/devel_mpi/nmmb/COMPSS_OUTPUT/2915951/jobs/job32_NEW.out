[JAVA EXECUTOR] executeTask - Begin task execution
WORKER - Parameters of execution:
  * Method type: BINARY
  * Method definition: [BINARY=ifort]
  * Parameter types: java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String java.lang.String
  * Parameter values: -mcmodel=large -shared-intel -convert big_endian -traceback -assume byterecl -O3 -fp-model precise -fp-stack-check -c /gpfs/projects/bsc19/bsc19533/NMMB-BSC/nmmb-bsc-ctm-v2.0/PREPROC/VARIABLE/module_flt.f90 -o /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_32/module_flt.o -module /gpfs/projects/bsc19/bsc19533/NMMB-BSC/nmmb-bsc-ctm-v2.0/PREPROC/VARIABLE/
  * Parameter streams: UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED
  * Parameter prefixes: null null null null null null null null null null null null null null null null null

[BINARY INVOKER] Begin binary call to ifort
[BINARY INVOKER] On WorkingDir : /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_32
[BINARY INVOKER] BINARY CMD: ifort -mcmodel=large -shared-intel -convert big_endian -traceback -assume byterecl -O3 -fp-model precise -fp-stack-check -c /gpfs/projects/bsc19/bsc19533/NMMB-BSC/nmmb-bsc-ctm-v2.0/PREPROC/VARIABLE/module_flt.f90 -o /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_32/module_flt.o -module /gpfs/projects/bsc19/bsc19533/NMMB-BSC/nmmb-bsc-ctm-v2.0/PREPROC/VARIABLE/ 
[BINARY INVOKER] Binary STDIN: null
[BINARY INVOKER] Binary STDOUT: null
[BINARY INVOKER] Binary STDERR: null
[BINARY EXECUTION WRAPPER] ------------------------------------
[BINARY EXECUTION WRAPPER] Executing binary command
[BINARY EXECUTION WRAPPER] ------------------------------------
[BINARY EXECUTION WRAPPER] CMD EXIT VALUE: 0
[BINARY EXECUTION WRAPPER] ------------------------------------
[BINARY EXECUTION WRAPPER] CMD OUTPUT:
[BINARY EXECUTION WRAPPER] ------------------------------------
[JAVA EXECUTOR] executeTask - End task execution
